WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -4,21 +4,21 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="title" content="Wasabi Wallet - Unfairly Private">
-    <meta name="description" content="Open-source, non-custodial, privacy-focused Bitcoin wallet, that implements trustless coin shuffling with mathematically provable anonymity - Chaumian CoinJoin.">
+    <meta name="description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for Desktop, that implements trustless CoinJoin over the Tor anonymity network.">
```

Comment:
If you are optimizing to keywords, wouldn't be better to write WasabiWallet? How is this working?

Reply:
You are right @molnard but I think users call Wasabi to WasabiWallet. Even we call it Wasabi.

Reply:
@molnard no,We are already in first place for those keywords and we don't need to work on them

Reply:
ACK

Reply:
```suggestion
    <meta name="description" content="Wasabi is an open-source, non-custodial, privacy-focused Bitcoin wallet for Desktop, it implements trustless CoinJoin over the Tor anonymity network.">
```